### PR TITLE
[12.0][MIG] fetchmail_incoming_log

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: fetchmail_incoming_log/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import test_fetchmail_incoming_log
```
Comment (tbaden) [inline fetchmail_incoming_log/tests/__init__.py]: missing newline

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM, just a really minor detail

File: fetchmail_incoming_log/tests/__init__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply fetchmail_incoming_log/tests/__init__.py]: added newline

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@OCA/server-environment-maintainers can we get this merged? Thanks!

**Comment by AaronHForgeFlow** [comment]
Ready to merge

**Pull request merged**